op-service/eth/types.go

```diff
@@ -204,6 +204,7 @@ type (
 )
 
 type ExecutionPayloadEnvelope struct {
+	EndOfSequencing       *bool             `json:"endOfSequencing,omitempty"`
```

Comment:
I guess we should also learn how to update https://github.com/taikoxyz/optimism/blob/taiko/op-service/eth/ssz.go#L466 accordingly and correctly, otherwise this `EndOfSequencing` won't be broadcasted?

Reply:
done + test